### ISSUE TITLE
Streamshare integration

### ISSUE THREAD

**Issue by 0PandaDEV** [0PandaDEV/Qopy#4]
Add a integration where you can directly upload a file or text you copied to https://streamshare.wireway.ch

**Comment by EarMaster** [comment]
Don't focus on a specific provider. Many tools allow sharing content by https request (e.g. timvisee/send or pglombardo/PasswordPusher). Maybe this should be configurable so everyone can use the provider of their choice (or even choose between several configured providers)...

**Comment by 0PandaDEV** [comment]
Yeah, I agree you this was just my first suggestion because a friend had the idea.
